.github/merge_request_template/Default.md

```diff
@@ -0,0 +1,26 @@
+## Describe your changes
+
+### JIRA link
+<!--- Please include a link to the JIRA ticket if there is one associated with -->
```

Comment:
NP : every PR requires a JIRA ticket, I would expand this to point a user to the contributing guide if they do not have a JIRA ticket for their change